pkg/kubelet/pleg/generic.go

```diff
@@ -275,6 +277,14 @@ func (g *GenericPLEG) relist() {
 				metrics.PLEGDiscardEvents.Inc()
 				klog.ErrorS(nil, "Event channel is full, discard this relist() cycle event")
 			}
+			// Log exit code of containers when they finished in a particular event
+			if err == nil {
+				for _, containerStatus := range status.ContainerStatuses {
+					if !containerStatus.FinishedAt.IsZero() && containerStatus.ID.ID == events[i].Data  {
```

Comment:
unnecessary duplicated space after `events[i].Data`.
Please remove it to pass pull-kubernetes-verify job.

Reply:
Fixed. Thanks